fares-data-build-tool/fdbt-site
# Feature/tfn 494 my account summary page

# Description

-   This PR introduces a new page to display a user's account details (email address, password and noc code).

# Testing instructions

-   Pull down this branch and run the site locally.
-   Sign in to the site and navigate to this page by clicking on the 'My Account' link. 
-   Validate that the page displays as expected and that your email address and noc code are displayed. Also check that the 'Change' button next to password navigates to the changePassword page (although this does not yet exist).

# Type of change


# Checklist:


[account.pdf](https://github.com/fares-data-build-tool/fdbt-site/files/4785557/account.pdf)
